src/panels/developer-tools/service/developer-tools-service.js

```diff
@@ -274,11 +279,17 @@ class HaPanelDevService extends PolymerElement {
     this.hass.callService(this._domain, this._service, this.parsedJSON);
   }
 
+  _fillExampleData() {
+    const example = this._attributes.reduce(
+      (acc, cur) => ({ ...acc, [cur.key]: cur.example }),
```

Comment:
This is so inefficient. You keep copying the object for each key in the object. Better to just loop over it.

Reply:
`for (const key of Object.keys(this._attributes)) { data[key] = this._attributes[key].example }`